definintion-of-iss-project.md

```diff
@@ -0,0 +1,58 @@
+# Title
+
+The Definition of an InnerSource Software (ISS) Project. We will largely take the ideas that follow
+from the definition of an Open Source Software (OSS) Project. Note, the definition of an OSS Project
+follows from [opensource.com's definition](https://opensource.com/resources/what-open-source). Further,
+[RedHat also supplies a defintion of OSS project](https://www.redhat.com/en/topics/open-source/what-is-open-source).
+
+Our goal here is to mimic the [open source way that opensource.com](https://opensource.com/open-source-way)
+provides. In doing this, we foster collaboration and community inside our enterprises.
+
+## Definition(s)
+
+The following are the components are essential for a project to be declared an InnerSource Software 
+project. Note, the distinction between Open Source and InnerSource is merely
+the network topology surrounding the source control management system instance. If the world at large
```

Comment:
I like this separation around network infra 😉. Depending on the audience it might be a bit technical.
What do you think about cases of e.g. an ISS project in a private repo on the public GitHub instance (this one right here)? Network-wise this would be available, access restrictions come in at a different layer.

Reply:
Good point there.